connectivity/FEATURE_BLE/source/pal/PalAttClientToGattClient.cpp

```diff
@@ -16,31 +16,32 @@
  * limitations under the License.
  */
 
-#include "ble/internal/PalGattClient.h"
-#include "ble/internal/PalAttClient.h"
+#include "source/pal/PalGattClient.h"
+#include "source/pal/PalAttClient.h"
+#include "source/pal/PalAttClientToGattClient.h"
 
 namespace ble {
 
-PalGattClient::PalGattClient(PalAttClient& client) : _event_handler(nullptr), _client(client)
+PalAttClientToGattClient::PalAttClientToGattClient(PalAttClient& client) : _event_handler(nullptr), _client(client)
```

Comment:
why the name change? is this not the PalGattClient? Is it relevant that it uses AttClient internally?

Reply:
No, it is not the `PalGattClient` interface, this is a generic **implementation** of the `PalGattClient` interface using a `PalAttClient`. I moved it outside of Cordio as it is independent from it.